### DISCUSSION TITLE
How do I use namespaces like with express?

### DISCUSSION THREAD

**Discussion by DoctaCloak** [pengzhanbo/vite-plugin-mock-dev-server#16] [Q&A] [answered]
I’d like to suggest this to my team but one question…say I have a route

/authentications/:uuid/init or /authentications/:uuid how would I support that with your plugin like I could in express or webpackdev server?

**Comment by pengzhanbo** [comment, marked as answer]
The plugin already supports this feature.

You can: 
```ts
import { defineMock } from 'vite-plugin-mock-dev-server'

export default defineMock({
  url: '/authentications/:uuid/init',
  body({ params }) {
      console.log(params.uuid)
  }
})
```
The `uuid` in the route will be resolved into the `request.params` object.

**Reply by DoctaCloak** [reply]
you’re a god ❤️

**Reply by pengzhanbo** [reply]
😊
